optimizely/helpers/condition.py

```diff
@@ -30,8 +31,7 @@ class ConditionalOperatorTypes(object):
 class ConditionEvaluator(object):
   """ Class encapsulating methods to be used in audience condition evaluation. """
 
-  def __init__(self, condition_data, attributes):
-    self.condition_data = condition_data
+  def __init__(self, attributes):
```

Comment:
can be instantiated with attributes, that is stored as instance variable. As this remains common for multiple audience evaluation. 
evaluate method accepts the condition structure